### DISCUSSION TITLE
2.1.1.Final

### DISCUSSION THREAD

**Discussion by chengfang** [jberet/jberet-jpa-repository#9] [Announcements]
## What's Changed
* JpaJobRepository by @amoscatelli in https://github.com/jberet/jberet-jpa-repository/pull/1
* upgrade jberet parent from 2.1.1.Final-SNAPSHOT to 2.1.1.Final by @chengfang in https://github.com/jberet/jberet-jpa-repository/pull/2
* moving to jakarta packages by @amoscatelli in https://github.com/jberet/jberet-jpa-repository/pull/5
* moving javax persistence timeout to jakarta persistence timeout by @amoscatelli in https://github.com/jberet/jberet-jpa-repository/pull/7
* add pom scm and url properties by @chengfang in https://github.com/jberet/jberet-jpa-repository/pull/8

## New Contributors
* @amoscatelli made their first contribution in https://github.com/jberet/jberet-jpa-repository/pull/1
* @chengfang made their first contribution in https://github.com/jberet/jberet-jpa-repository/pull/2

**Full Changelog**: https://github.com/jberet/jberet-jpa-repository/commits/2.1.1.Final

<hr /><em>This discussion was created from the release <a href='https://github.com/jberet/jberet-jpa-repository/releases/tag/2.1.1.Final'>2.1.1.Final</a>.</em>

**Comment by amoscatelli** [comment]
Thank you @chengfang !
Is this automatically published on public maven repository also ?

**Comment by chengfang** [comment]
yes, it's available at https://repository.jboss.org/nexus/content/groups/public/, and will be sync'ed to maven central shortly.
